src/modules/completedCounter.js

```diff
@@ -0,0 +1,17 @@
+const completedCounter = () => {
+  const counterIcon = document.querySelector('#title-box__counter-icon');
+  let taskList = JSON.parse(localStorage.getItem('taskListData'));
+  taskList = taskList.filter((task) => task.completed === true);
+
+  counterIcon.classList = '';
+
+  if (taskList.length === 0) {
+    counterIcon.classList = '';
+  } else if (taskList.length >= 10) {
+    counterIcon.classList = 'bi bi-exclamation-circle-fill';
+  } else {
+    counterIcon.classList = `bi bi-${taskList.length}-circle-fill`;
+  }
+};
+
+export default completedCounter;
```

Comment:
Good job! A nice feature to count the number of completed tasks 💯🤓

![image](https://github.com/grauJavier/to-do-list/assets/78251927/d8da3735-2bab-4988-8938-75a5a767eb50)